Process Monitors (#5576)
DataDog/documentation: content/en/monitors/monitor_types/process.md

  text: "Schedule a downtime to mute a monitor"
- link: "monitors/monitor_status"
  tag: "Documentation"
  text: "Check your monitor status"
---

## Overview

> Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the behavior of any group of processes across hosts or tags.

Reviewer: ```suggestion
Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the behavior of any group of processes across your hosts or tags.
```

> Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the behavior of any group of processes across your hosts or tags.

Reviewer 2: "on the behavior" might be a little bit misleading as we only allow users to warn/alert based on the count of processes.

> Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the behavior of any group of processes across your hosts or tags.

Author: ```suggestion
Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the count of any group of processes across your hosts or tags.
```

## Monitor creation

To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.

### Select processes

> Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Use tags to refine your monitor to a specific scope.

Reviewer 2: we also support AND/OR/NOT operators in the query string.

https://docs.datadoghq.com/graphing/infrastructure/process/?tab=linuxwindows#search-syntax

Matching processes and counts are displayed below the search:

{{< img src="monitors/monitor_types/process/select_processes.png" alt="Select processes" responsive="true" style="width:90%;">}}

After defining your search, a graph is displayed above the search inputs with the total number of processes found.

#### Alert grouping

> `Simple Alert` (default): aggregates alerts over all reporting sources. You receive one alert when the aggregated value meets the set conditions.

Reviewer: ```suggestion
`Simple Alert` (default): aggregates alerts over all reporting sources. You receive one notification when the aggregated value meets the set conditions.
```

Author: I think alert is a better word here since people can set up monitors that don't send out notifications.

> `Multi Alert`: applies the alert to each source according to your group parameters. You receive an alert for each group that meets the set conditions.

Reviewer: ```suggestion
`Multi Alert`: applies the alert to each source according to your group parameters. You receive a notification for each group that meets the set conditions.
```
